server/src/services/metadata.service.ts

```diff
@@ -589,9 +589,18 @@ export class MetadataService extends BaseService {
     let dateTimeOriginal = dateTime?.toDate();
     let localDateTime = dateTime?.toDateTime().setZone('UTC', { keepLocalTime: true }).toJSDate();
     if (!localDateTime || !dateTimeOriginal) {
-      this.logger.warn(`Asset ${asset.id} has no valid date, falling back to asset.fileCreatedAt`);
-      dateTimeOriginal = asset.fileCreatedAt;
-      localDateTime = asset.fileCreatedAt;
+      // When a file is copied (but not moved) before being uploaded to immich, the target file creation
```

Comment:
We don't need a lots of comments here, maybe just

```
// use the earliest date as the fallback date
```

Reply:
So I'll remove the comment, as the variable `earliestDate` is quite explicit

Reply:
(change reverted)